### PR TITLE
Routing basics

### PR DESCRIPTION
**Pull request by eviera21**

### CODE REVIEW

File: frontend/src/Models.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from "react";
+
+const ModelsContext = React.createContext({
+  models: [], fetchModels: () => {}
+})
+export default function Models(){
+  const [models, setModels] = useState([])
+    const fetchModels = async () => {
+      const response = await fetch("http://localhost:8000/")
```
Comment (FernandoDavis) [inline frontend/src/Models.js:9]: Could use a config variable here instead of localhost to avoid losing track of the requests

**Review by FernandoDavis — APPROVED** [review]
LGTM, check the comment I left for further implementations on this code

**Comment by carolinasantia2** [comment]
LGTM

**Review by carolinasantia2 — COMMENTED** [review]
LGTM

**Pull request merged**